docs/api/cozy-client/classes/HasMany.md

```
@@ -228,7 +228,7 @@ to do that, you can use .data.length.
 
 *Defined in*
 
-[packages/cozy-client/src/associations/HasMany.js:93](https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/associations/HasMany.js#L93)
+[packages/cozy-client/src/associations/HasMany.js:93](https://github.com/linagora/cozy-client/blob/master/packages/cozy-client/src/associations/HasMany.js#L93)
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)